src/components/ui/Hero/Hero.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export interface HeroProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  children: React.ReactNode
+}
+
+const Hero = ({ testId = 'store-hero', children }: HeroProps) => {
```

Comment:
I think it's important to keep the `forwardRef` and `...otherProps` patterns in this component as well!